src/main/java/io/sprucehill/zalando/api/nativecart/service/AddressService.java

```diff
@@ -0,0 +1,77 @@
+package io.sprucehill.zalando.api.nativecart.service;
+
+import java.io.UnsupportedEncodingException;
+import javax.annotation.PostConstruct;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.entity.StringEntity;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import io.sprucehill.zalando.api.nativecart.model.AddressCheckRequest;
+import io.sprucehill.zalando.api.nativecart.model.AddressCheckResponse;
+import io.sprucehill.zalando.api.nativecart.model.CreateAddressRequest;
+import io.sprucehill.zalando.api.nativecart.model.CreateAddressResponse;
+import io.sprucehill.zalando.api.nativecart.model.GetAddressResponse;
+import io.sprucehill.zalando.api.nativecart.model.GetAddressesResponse;
+import io.sprucehill.zalando.api.nativecart.model.UpdateAddressRequest;
+import io.sprucehill.zalando.api.nativecart.model.UpdateAddressResponse;
+
+/**
+ * 
+ * @author dipteewarudkar
+ *
+ */
+public class AddressService extends AbstractService implements IAddressService{
+
+	@PostConstruct
+	@Override
+	public void postConstruct() {
```

Comment:
I am not really imagine the changes as of now in the AbstractService, i would remove it for now ,when we have a testable version of APIs i will merge/work on this case.